frontend/src/components/userProfile/AssignedStoryTranslationsTable.tsx

```diff
@@ -280,12 +280,12 @@ const AssignedStoryTranslationsTable = ({
   const generateStoryTranslationLink = (translation: StoryTranslation) => {
     const suffix = `${translation.storyId}/${translation.storyTranslationId}`;
     if (isAdmin) {
-      return `/story/${suffix}`;
+      return `#/story/${suffix}`;
```

Comment:
Is this link supposed to be `story/storyId/storyTranslationId`? I seem to be getting a 404 here but not sure what changed